pkgs/development/tools/summon/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, buildGoModule, fetchFromGitHub, lib, patchResolver ? true }:
+
+with stdenv.lib;
+
+buildGoModule rec {
+  pname = "summon";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "cyberark";
+    repo = "summon";
+    rev = "v${version}";
+    sha256 = "150vrivrnzdskf8c4vxacqyl67phmmx3vn798pvc6kdjnm829hnn";
+  };
+
+  vendorSha256 = "1597vrs4b7k6gkmkvf7xnd38rvjixmlcz0j7npmik9nbkm57l74m";
+
+  subPackages = [ "cmd" ];
+
+  patches = optional patchResolver [ ./resolve-paths.patch ];
```

Comment:
Does this need to be optional? I'm not sure, what do you think @tomberek 
Also it would be nice with a short comment explaining why the patch is there.

Reply:
My reasoning to make it optional was that it changes the intended behavior of the tool, which may, theoretically, break someone's workflow.